src/test/Rets_Api.postman_collection.json

```diff
@@ -45,9 +45,53 @@
 		},
 		{
 			"name": "Get by Filter",
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
 			"request": {
 				"method": "GET",
 				"header": [],
+				"body": {
```

Comment:
Please update the collection, we don't have a body at get request anymore. To be easier, try to use the qa collection as basis in postman, adjust the values and export the collection to the project folder ;)